src/gui/optionsdialog.cpp

```diff
@@ -489,6 +492,36 @@ void OptionsDialog::initializeLanguageCombo()
     }
 }
 
+
+
+void OptionsDialog::initializeThemeCombo()
+{
+    m_ui->comboTheme->addItem(tr("Default"));
+    if (UIThemeManager::instance()->isCustomAvailable())
+        m_ui->comboTheme->addItem(tr("Custom"));
+    m_ui->comboTheme->insertSeparator(m_ui->comboTheme->count());
+    m_ui->comboTheme->addItem(tr("Select..."));
+    m_ui->comboTheme->setCurrentIndex(Preferences::instance()->UIThemeSelection());
+
+    connect(m_ui->comboTheme, qOverload<int>(&QComboBox::currentIndexChanged), [&](int index)
+    {
+        if (index == (m_ui->comboTheme->count() - 1)) {
+            m_rccFilePath = QFileDialog::getOpenFileName(this, tr("Select Rcc File"), "", tr("Theme Resource File (*.rcc)"));
```

Comment:
It's good idea (and widely used practice) to use custom format name and file extension in such case, IMO. We don't expect any RCC file here but some preformatted content packed in it.

Reply:
I don't have any problem, suggestions?

Reply:
"Select qBittorrent theme file" for message.
.qbtheme or something like it for file extension.